game/src/main/java/io/castles/game/GameLobby.java

```diff
@@ -44,6 +47,26 @@ public void removePlayer(Player player) {
         }
     }
 
+    public void removePlayer(UUID playerId) {
+        Player playerToRemove = getPlayerById(playerId);
+        boolean removed = this.players.remove(playerToRemove);
+        if (!removed) {
+            throw new IllegalArgumentException(String.format("Player %s was not found in the list of players %s", playerToRemove, players));
+        }
```

Comment:
```suggestion
        removePlayer(playerToRemove)
```
Either this or remove the other `removePlayer` method.